src/components/RadioButton/RadioButton.types.ts

```diff
@@ -3,5 +3,5 @@ import { ReactNode } from 'react';
 import { DefaultComponentProps, HTMLInputAttrs } from '../../types';
 
 export interface RadioButtonProps extends DefaultComponentProps, HTMLInputAttrs {
-	label: string | ReactNode;
+	label?: ReactNode;
```

Comment:
Why did you make this stricter?
aren't there use cases where string is also possible?

Reply:
ReactNode covers string types so it's still the same as before